s3upload.js

```diff
@@ -46,6 +48,10 @@ function S3Upload(options) {
     this.handleFileSelect(files);
 }
 
+function getFileMimeType(file) {
+    return mime.lookup(file.name);
```

Comment:
Since `file.type` might be available, should we instead use:

    file.type || mime.lookup(file.name);

Perhaps?